stock_custom_exchange_rate_date/views/stock_picking_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form.inherit.custom.rate.date</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='scheduled_date']" position="before">
+                <field name="exchange_rate_date"/>
```

Comment:
this field only make sense when it is a reception only should be in picking_type = supplier

Reply:
Oh really? I wasn't aware of that, I though this also made sense on outgoing transfers.

Reply:
Done, readonly domain added